README.md

```diff
@@ -38,7 +38,7 @@ Below is an example for scanning `CRITICAL` vulnerabilities.
 steps:
   - command: ls
     plugins:
-      - trivy#v1.8.0:
+      - trivy#v1.9.0:
 	severity: "CRITICAL"
```

Comment:
Also,Just I made change with same commit and updated `README.md` , Please can you look this too.

Reply:
So, I think I see the issue now. It looks like the line `severity: "CRITICAL"` is using a tab character rather than spaces. Can you replace the tabs (the red blocks in this diff) with the proper amount of spaces?

Reply:
Yeah....that's wright, It's fixed now.

Reply:
I just noticed the REAMDE is basically a copy of the template... so I am not sure how is supposed to work.

We can address this in a separate PR.